Simplenote.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "SimplenoteFoundation",
+        "repositoryURL": "git@github.com:Automattic/SimplenoteFoundation-Swift.git",
+        "state": {
+          "branch": "issue/visibility-modifiers",
```

Comment:
Do we want this pointing at a branch in `develop`? Or should it perhaps be versioned like a Pod?

Reply:
Oh! It'll be pointing to actual released versions. It's mapping towards a specific branch for testing purposes, I'll switch before merging.

Thank you for the heads up sir!